code/datums/gamemode/factions/faction.dm

```diff
@@ -81,7 +81,7 @@ var/list/factions_with_hud_icons = list()
 /datum/faction/proc/HandleRecruitedMind(var/datum/mind/M)
 	for(var/datum/role/R in members)
 		if(R.antag == M)
-			WARNING("Mind was already a role in this faction")
+			warning("Mind was already a role in this faction")
```

Comment:
Why would you change these from `WARNING` (which shows file, line number, src, usr) to `warning` (which only shows the message)?